src/api/app/presenters/notification_presenter.rb

```diff
@@ -45,4 +34,18 @@ def excerpt
             end
     text.to_s.truncate(100)
   end
+
+  def kind_of_request
+    return unless @model.notifiable_type == 'BsRequest'
+
+    request = @model.notifiable
+    return "Multiple Actions on project #{request.bs_request_actions.first.target_project}" if request.bs_request_actions.size > 1
+
+    BsRequest.actions_summary(@model.event_payload)
+  end
+
+  def commenters
```

Comment:
This you can do with the commentable directly no? Like `commentable.comments.where('updated_at >= ?', @model.unread_date)...`

Sounds like an additional `commenters` scope to me.

Reply:
True! Changed!

Reply:
I meant scope inside the `Project`, `Package` or `BsRequest` but okay. Something for later I guess...